### PR TITLE
Reapply the change from #97

### PR DESCRIPTION
**Pull request by kristiandupont**
..which was lost somewhere in the schema updates.

### CODE REVIEW

**Comment by bdash** [comment]
Any chance that you can add a test so that this can't break in the same way in the future?

File: src/object_store.cpp [hunk 1]
```diff
@@ -250,7 +250,7 @@ StringData ObjectStore::object_type_for_table_name(StringData table_name) {
 }
 
 std::string ObjectStore::table_name_for_object_type(StringData object_type) {
-    return std::string(c_object_table_prefix) + object_type.data();
+    return std::string(c_object_table_prefix) + static_cast<std::string>(object_type);
```
Comment (bdash) [inline src/object_store.cpp]: Seems odd to use `static_cast<std::string>(…)` when we're doing`std::string(…)` on the same line of code.

Comment (kristiandupont) [inline src/object_store.cpp]: Good point, I'll align that.

**Comment by kristiandupont** [comment]
@bdash is this ok?

**Comment by bdash** [comment]
👍

Please squash down to one commit prior to merging.

**Pull request merged**